src/main/java/Menu.java

```diff
@@ -0,0 +1,123 @@
+                                        System.out.println("success");
+                                    } else {
+                                        System.out.println("Not enough money!");
+                                    }
+
+                                } else {
+                                    System.out.println("Such a card does not exist.");
+                                }
+                            } else {
+                                System.out.println("Probably you made a mistake in the card number. Please try again!");
+                            }
+                        } else {
+                            System.out.println("You can't transfer money to the same account!");
+                        }
+
+                        break;
+                    case 4:
+                        DBConnector.delAccount(id);
+                        System.out.println("The account has been closed!");
+                        loggedIn = false;
+                        id = -1;
+                        break;
+                    case 5:
+                        break;
+                    case 0:
+                        System.out.println("Bye!");
+                        exit(0);
+                    default:
+                        System.out.println("Wrong number");
+                        break;
+                }
+
+            } else {
+                switch (choose) {
+                    case 1:
+                        Account account = new Account();
+                        System.out.println("Your card has been created");
+                        System.out.println("Your card number:");
+                        System.out.println(account.getCardNumber());
+                        System.out.println("Your card PIN:");
+                        System.out.println(account.getPIN());
+                        break;
+                    case 2:
+                        System.out.println("Enter your card number:");
+                        String number = scanner.nextLine();
+                        number = scanner.nextLine();
+                        System.out.println("Enter your PIN:");
+                        int PIN = scanner.nextInt();
+                        int logId = Account.login(number, PIN);
+                        if (logId == -1) {
+                            System.out.println("Wrong card number or PIN");
+                        } else {
+                            System.out.println("You have successfully logged in");
+                            loggedIn = true;
+                            id = logId;
+                        }
+                        break;
+                    case 0:
+                        System.out.println("Bye!");
+                        exit(0);
+                    default:
+                        System.out.println("Wrong number");
+                        break;
+                }
+            }
+        }
+    }
```

Comment:
This method is too big and unreadable. Can you refactor it?

Reply:
fixed